ddtrace/contrib/trace_utils.py

```diff
@@ -194,13 +194,20 @@ def set_http_meta(
         span._set_str_tag(http.RETRIES_REMAIN, retries_remain)
 
 
-def activate_distributed_headers(tracer, int_config=None, request_headers=None):
+def activate_distributed_headers(tracer, int_config=None, request_headers=None, override_distributed_tracing=None):
```

Comment:
```suggestion
def activate_distributed_headers(tracer, int_config=None, request_headers=None, override=None):
```

I think we can shorten this for usability :)